src/applications/healthcare/questionnaire/components/get-help/GetHelpFooter.jsx

```diff
@@ -1,28 +1,21 @@
 import React from 'react';
-import Telephone, {
-  CONTACTS,
-} from '@department-of-veterans-affairs/formation-react/Telephone';
+import Telephone from '@department-of-veterans-affairs/formation-react/Telephone';
 
 export default function GetHelpFooter() {
+  const HELP_NUMBER = '844-698-2311';
+  const TTY_NUMBER = '711';
   return (
     <div className="row questionnaire-help-footer">
       <div className="usa-width-two-thirds medium-8 columns">
         <h2 className="help-heading">Need help?</h2>
         <p>
-          For help filling out this form, you can contact your local
-          coordinator, or call our main VA information line at{' '}
-          <Telephone contact={CONTACTS.CAREGIVER} />.
+          For help filling out this form, or if the form isn’t working right,
+          please call the VA.gov Technical Help Desk at{' '}
+          <Telephone contact={HELP_NUMBER} />.
         </p>
         <p>
-          To report a problem with this form, please call the VA.gov Technical
-          Help Desk at <Telephone contact={CONTACTS.HELP_DESK} />.
+          <Telephone contact={TTY_NUMBER}>TTY: {TTY_NUMBER}</Telephone>
```

Comment:
Is the TTY_NUMBER same as the FEDERAL_RELAY_SERVICE?

Reply:
It is not, the TTY is `711`, while the `FEDERAL_RELAY_SERVICE`is `8008778339`